apiserver/common/networkingcommon/networkconfigapi_test.go

```diff
@@ -4,91 +4,389 @@
 package networkingcommon_test
 
 import (
+	"github.com/golang/mock/gomock"
+	"github.com/juju/errors"
 	jc "github.com/juju/testing/checkers"
 	gc "gopkg.in/check.v1"
+	"gopkg.in/juju/names.v2"
+	"gopkg.in/mgo.v2/txn"
 
-	"github.com/juju/juju/apiserver/common"
 	"github.com/juju/juju/apiserver/common/networkingcommon"
+	"github.com/juju/juju/apiserver/common/networkingcommon/mocks"
 	"github.com/juju/juju/apiserver/params"
-	jujutesting "github.com/juju/juju/juju/testing"
+	"github.com/juju/juju/core/network"
 	"github.com/juju/juju/state"
 )
 
+// modelOpStub is a no-op model operation that can store the
```

Comment:
Minor unimportant nit; that would be a recorder/spy ;-)

Reply:
Changed the name.